core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -764,6 +764,22 @@ private SqlLibraryOperators() {
           ReturnTypes.BIGINT_NULLABLE, OperandTypes.TIMESTAMP,
           SqlFunctionCategory.TIMEDATE);
 
+  /** BigQuery's "DATETIME_ADD(timestamp, interval) function; Behaves similarly
```

Comment:
I wouldn't describe it as 'BigQuery's'. Other DBs might support it one day.

The "because in Calcite, datetime is a type alias for timestamp" is confusing, and I'd remove it. The fact is, both accept TIMESTAMP or TIMESTAMP WITH LOCAL TIME ZONE arguments, and both return the same type as their first argument.